packages/ui/modals/Service.tsx

```diff
@@ -1,14 +1,198 @@
+			(subsections, { attribute, supplement }) => {
+				const { tsKey, categories } = attribute
+				const { category } = categories.at(0)!
+				const { tag } = category
+
+				switch (tag) {
+					case 'languages': {
+						supplement.forEach(({ language }) => {
+							const { nativeName, languageName } = language as Language
+							subsections[tag].push(languageName)
+						})
+						break
+					}
+					case 'community': {
+						subsections[tag].push({ tsKey, icon: 'a', variant: tag })
+						break
+					}
+					case 'cost': {
+						subsections[tag] = tsKey
+						break
+					}
+					case 'srvFocus': {
+						subsections[tag].push(t(tsKey))
+						break
+					}
+					case 'eligibility': {
+						tsKey === 'elig-age'
+							? (subsections[tag]['age'] = tsKey)
+							: subsections[tag]['requirements'].push(t(tsKey))
+						break
+					}
+					case 'misc': {
+						subsections[tag].push(t(tsKey))
+						break
+					}
+				}
+				return subsections
+			},
+			attributeCategories
+		)
+
+		return (
+			<Stack spacing={24} className={classes.modalBody}>
+				<div>{allServices}</div>
+				<SectionDivider title='get-help'>
+					<Subsection title={`direct-phone${directPhones.length > 1 ? 's' : ''}`}>
+						<Text>{directPhones}</Text>
+					</Subsection>
+					<Subsection title='direct-email'>
+						<Text>{directEmail ? directEmail.email.email : 'not available'}</Text>
+					</Subsection>
+					<Subsection title='direct-website'>
+						<Text>{/* todo: get data for website */}</Text>
+					</Subsection>
+					<Subsection title='hours'>
+						<>
+							<Text>{t('service-default-timezone')}</Text>
+							{serviceHours}
+						</>
+					</Subsection>
+				</SectionDivider>
+				<SectionDivider title='clients-served'>
+					<Subsection title='community-focus'>
+						<BadgeGroup badges={community} withSeparator={false} />
+					</Subsection>
+					<Subsection title='target-population'>
+						<UL>{srvFocus}</UL>
+					</Subsection>
+				</SectionDivider>
+				<SectionDivider title='cost'>
+					<Subsection>
+						<Group spacing='sm'>
+							<Icon icon='carbon:piggy-bank' />
+							<Text>{cost.length > 0 || 'N/A'}</Text>
+						</Group>
+					</Subsection>
+					{/* todo: ask where to look for cost details data */}
+				</SectionDivider>
+				<SectionDivider title='elegibility-requirements'>
+					<Subsection title='eligible-age'>
+						<Text>{eligibility.age ? `${eligibility.age} and older` : 'not applicable'}</Text>
+					</Subsection>
+					<Subsection title='requirements'>
+						<UL>{eligibility.requirements}</UL>
+					</Subsection>
+				</SectionDivider>
+				<SectionDivider title='languages'>
+					<Subsection title='languages'>
+						<UL>{languages}</UL>
+					</Subsection>
+				</SectionDivider>
+				<SectionDivider title='additional-information'>
+					<Subsection title='Additional information'>
+						<UL>{misc}</UL>
+					</Subsection>
+				</SectionDivider>
```

Comment:
These sections will be generated based off of which "attributes" are attached to the service record. Each one of the section headers corresponds with one of the "attribute categories." [This seed file](https://github.com/weareinreach/InReach/blob/272588700cdf81dd2c55dda3e5471e9b7ef1f71f/packages/db/seed/data/09-attributes.ts#L1) will give some additional context to how this is set up. As far as the icons - the icon key (the string that would be fed in to the `icon` prop) will be associated with the attribute record. Additional context for an attribute will be attached via a "supplement" record (example - the languages section would be the 'lang-offered' attribute with multiple Language records attached). [This part of the data schema](https://github.com/weareinreach/InReach/blob/272588700cdf81dd2c55dda3e5471e9b7ef1f71f/packages/db/prisma/schema.prisma#L885-L981) shows the structure.